README.md

```diff
@@ -157,7 +157,7 @@ timestamp = source_anchor.signed_timestamp
 origin_server_certs = source_anchor.origin_server_certs
 ```
 
-If you have chosen `Verify Condition` on the [Yoti Dashboard](https://www.yoti.com/dashboard/applications) with the age condition of "Over 18", you can retrieve the user information as follows:
+If you have chosen `Verify Condition` on the [Yoti Hub](https://hub.yoti.com/applications) with the age condition of "Over 18", you can retrieve the user information as follows:
```

Comment:
I think `/applications` behaves differently for organisations - I changed this to just `https://hub.yoti.com` in other SDKs - thoughts @echarrod ?

Reply:
Yep, `/applications` is personal applications, let's drop it